phoenix-core/src/main/antlr3/PhoenixSQL.g

```diff
@@ -808,7 +808,14 @@ bind_expression  returns [BindParseNode ret]
     ;
     
 value_expression returns [ParseNode ret]
-    :   i=add_expression { $ret = i; }
+    :   i=pre_value_expression { $ret = i; }
```

Comment:
@petercdc  Is there a reason you added some of the operators here and the other operators in array_expression ?